code/src/patches.s

```diff
@@ -1453,15 +1453,10 @@ TurboTextClose_patch:
 TurboTextSignalNPC_patch:
     bl hook_TurboTextSignalNPC
 
-.section .patch_SkipSongReplayForTimeBlocksOne
-.global SkipSongReplayForTimeBlocksOne_patch
-SkipSongReplayForTimeBlocksOne_patch:
-    b hook_SkipSongReplayForTimeBlocksOne
-
-.section .patch_SkipSongReplayForTimeBlocksTwo
-.global SkipSongReplayForTimeBlocksTwo_patch
-SkipSongReplayForTimeBlocksTwo_patch:
-    b hook_SkipSongReplayForTimeBlocksTwo
+.section .patch_SkipSongReplays_TimeBlocksFix
+.global SkipSongReplays_TimeBlocksFix_patch
+SkipSongReplays_TimeBlocksFix_patch:
+    moveq r1,#0x1
```

Comment:
This should only be replaced if `Skip Song Replays` is enabled, right? Otherwise it changes vanilla behaviour, which some may not like.

Reply:
Thanks, I somehow totally forgot about that.